modules/015-admission-policy-engine/crds/security-policy.yaml

```diff
@@ -458,6 +458,12 @@ spec:
                             description: The list of Cosign compliant public keys.
                             items:
                               type: string
+                          registryCAs:
```

Comment:
I don't think we need an array here.
Everywhere we have a `ca` field with a single value
```
ca:
  type: string
```
I think we have to follow this pattern here  to have a common flow

Reply:
disagree, but whatever